src/hooks/useNfc.ts

```diff
@@ -49,7 +70,7 @@ export const useNfc = () => {
   }, []);
 
   const readingNfcLoop = useCallback(
-    async (pr: string) => {
+    async (pr: string, amount: number) => {
```

Comment:
I had to pass the amount into this function to check if pin is required.

It is also needed for the commented out part, the lnurlp request as far as I understand. Was there a reason why it is commented out? Are there any bugs or is it a bad idea to add this parameter to the function?